controller/monitor/manager.py

```diff
@@ -0,0 +1,42 @@
+from typing import Any, List
+from submodules.model.business_objects import monitor
```

Comment:
monitor is ambiguous imo. Maybe task_monitor?

Reply:
done